src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.scss

```diff
@@ -45,7 +45,7 @@ $header-weight: 500;
       font-weight: $header-weight;
       padding: 16px;
       text-align: left;
-      border-bottom: 1px solid #cacaca;
+      border-bottom: 1px solid #8c8c8c;
       background-color: #ededed;
```

Comment:
seems redundant. should inherit from thead?

Reply:
modified